pkg/server/serverpb/admin.proto

```diff
@@ -424,11 +424,14 @@ message SettingsResponse {
 
 // HealthRequest inquires whether the addressed node is healthy.
 message HealthRequest {
```

Comment:
Let's be specific about what this means since there was all of this ambiguity over in https://github.com/cockroachdb/cockroach/issues/44832. Something like:

HealthRequest requests a liveness or readiness check.

A liveness check is triggered via `ready` set to `false`. In this mode, an empty response is returned immediately, that is, the caller merely learns that the process is running.

A readiness check (`ready == true`)  is suitable for determining whether user traffic should be directed at a given node, for example by a load balancer. In this mode, a successful response is returned only if the node
- is not in the process of shutting down or booting up (including waiting for cluster bootstrap);
- is regarded as healthy by the cluster via the recent broadcast of a liveness beacon.
Absent either of these conditions, an error code will result.

Reply:
PS if you know of a better place to describe this, I'm up for that too. Definitely also something we may want to carry over to docs once we've worked out the verbiage.